scripts/proteinAtlas/parse_protein_atlas.py

```diff
@@ -0,0 +1,227 @@
+    return uniprot_to_dcid
+
+
+def get_gene_to_dcid_list(gene_to_uniprot_list, uniprot_to_dcid):
+    """
+    Args:
+        gene_to_uniprot_list: a dic mapping gene to a list of UniProt entry
+        uniprot_to_dcid: a dic mapping UniProt entry to Data Commons DCID
+    Returns:
+        A dict mapping gene to a list of protein DCID in Data Commons.
+        example: {'TSPAN6': ['TSN6_HUMAN']}
+    """
+    gene_to_dcid_list = {}
+    # This for loop generate the mapping
+    for gene in gene_to_uniprot_list:
+        # One gene can map to several UniProt entry
+        uniprot_list = gene_to_uniprot_list[gene]
+        dcid_list = []
+        for uniprot in uniprot_list:
+            dcid_list.append(uniprot_to_dcid[uniprot])
+        gene_to_dcid_list[gene] = dcid_list
+    return gene_to_dcid_list
+
+
+def get_class_name(a_string):
+    """Convert a name string to format: ThisIsAnUnusualName.
+    Take a space delimited string, return a class name such as ThisIsAnUnusualName
+    Here we use this function for instance name. Thus it allows to start with a number
+    """
+    joint_name = a_string.title().replace(' ', '')
+    # substitute except for  _, character, number
+    non_legitimate = re.compile(r'[\W]+')
+    class_name = non_legitimate.sub('', joint_name)
+    return class_name
+
+
+def generate_mcf(protein_dcid, tissue, cell, expression, reliability):
+    """generate a data mcf"""
+    name = '_'.join([protein_dcid, tissue, cell])
+    mcf = ('Node: dcid:bio/' + name + '\n' + 'typeOf: HumanProteinOccurrence' +
+           '\n' + 'name: "' + name + '"' + '\n' + 'detectedProtein: dcs:bio/' +
+           protein_dcid + '\n' + 'humanTissue: dcs:' + tissue + '\n' +
+           'humanCellType: dcs:' + cell + '\n' +
+           'proteinExpressionScore: dcs:' + expression + '\n' +
+           'humanProteinOccurrenceReliability: dcs:' + reliability)
+    return mcf
+
+
+def mcf_from_row(row, expression_map, reliability_map, gene_to_dcid_list):
+    """Generate data mcf from each row of the dataframe"""
+    gene = row['Gene name']
+    tissue = get_class_name(row['Tissue'])
+    cell = get_class_name(row['Cell type'])
+    expression = expression_map[row['Level']]
+    reliability = reliability_map[row['Reliability']]
+    if gene not in gene_to_dcid_list:
+        # skip case when there is no gene to dcid mapping
+        return None
+    dcid_list = gene_to_dcid_list[gene]
+
+    mcf_list = []
+    for protein_dcid in dcid_list:
+        mcf_list.append(
+            generate_mcf(protein_dcid, tissue, cell, expression, reliability))
+    return '\n\n'.join(mcf_list)
+
+
+def get_tissue_enum(tissue):
+    """Generate a enum instance for a tissue"""
+    name = get_class_name(tissue)
+    mcf = ('Node: dcid:' + name + '\n' + 'typeOf: dcs:HumanTissueEnum' + '\n' +
+           'name: "' + name + '"\n' + 'description: "' + tissue[0].upper() +
+           tissue[1:] + '"\n' + 'domainIncludes: dcs:HumanTissueEnum\n')
```

Comment:
readability comment from the other PR applies. i.e., have a list and join by '\n'. Or use template.

Reply:
Changed. Nice suggestion!